src/main/java/com/codeofus/rent_a_park/models/Spot.java

```diff
@@ -13,7 +15,7 @@
 @NoArgsConstructor
 @ToString
 @FieldDefaults(level = AccessLevel.PRIVATE)
-public class Spot {
+public class Spot implements Serializable {
```

Comment:
Why do we need "implements Serializable"?

Reply:
in this case we don't need it because of no caching on it, you are right, might be left accidentally